src/main/java/com/zaxxer/hikari/HikariConfig.java

```diff
@@ -53,6 +53,7 @@ public class HikariConfig implements HikariConfigMXBean
    private static final long SOFT_TIMEOUT_FLOOR = Long.getLong("com.zaxxer.hikari.timeoutMs.floor", 250L);
    private static final long IDLE_TIMEOUT = MINUTES.toMillis(10);
    private static final long MAX_LIFETIME = MINUTES.toMillis(30);
+   private static final double MAX_LIFETIME_VARIANCE = 2.5;
```

Comment:
Maybe this constant should be called `DEFAULT_MAX_LIFETIME_VARIANCE` instead? As it appears to be the default value.

Reply:
Great suggestion, went ahead and changed it.

Reply:
I don't want to give you false expectations that your PR will be merged soon. I don't have push permissions. I'm just an open source engineer happening to look at this PR, because I want to see how reactive the maintainers are.